### ISSUE TITLE
Can't reproduce the result

### ISSUE THREAD

**Issue by OStars** [PlusLabNLP/DEGREE#10]
Hello, I tried to run the code on ACE05-E+ with your default end-to-end config, but finally, the arg_cls f1 was 54.94 while the arg_cls reported in the paper was 56.3. Is that because of the hyperparameters I use or the random seed?
![image](https://user-images.githubusercontent.com/57997884/226545429-94fd27d2-20f8-4ccb-9af6-dcb0718178b1.png)

**Comment by ej0cl6** [comment]
Thanks for your interest in our work. Sometimes there will be some randomness. Maybe you can try to train it with different random seeds.
